skimage/transform/hough_transform.py

```diff
@@ -148,6 +148,27 @@ def hough_circle(image, radius, normalize=True, full_output=False):
         Hough transform accumulator for each radius.
         R designates the larger radius if full_output is True.
         Otherwise, R = 0.
+
+    Examples
+    --------
+    >>> from skimage.transform import hough_circle
+    >>> img = np.zeros((100,100),dtype=np.bool_)
```

Comment:
There should be a whitespace after a comma. I was telling you that there would be some nitpicking :-).